apps/chat/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,77 @@
+import { IconMessage2 } from '@tabler/icons-react';
+import { useCallback } from 'react';
+
+import { useRouter } from 'next/router';
+
+import classNames from 'classnames';
+
+import { useTranslation } from '@/src/hooks/useTranslation';
+
+import { Translation } from '@/src/types/translation';
+
+import { UISelectors } from '../../store/ui/ui.selectors';
+import { ConversationsActions } from '@/src/store/conversations/conversations.reducers';
+import { ConversationsSelectors } from '@/src/store/conversations/conversations.selectors';
+import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
+import { SettingsSelectors } from '@/src/store/settings/settings.selectors';
+
+import { DEFAULT_CONVERSATION_NAME } from '../../constants/default-ui-settings';
```

Comment:
maybe we can try to use same import style
```suggestion
import { UISelectors } from '@/src/store/ui/ui.selectors';
import { ConversationsActions } from '@/src/store/conversations/conversations.reducers';
import { ConversationsSelectors } from '@/src/store/conversations/conversations.selectors';
import { useAppDispatch, useAppSelector } from '@/src/store/hooks';
import { SettingsSelectors } from '@/src/store/settings/settings.selectors';

import { DEFAULT_CONVERSATION_NAME } from '@src/constants/default-ui-settings';
```